### PR TITLE
Schemas updated to run Sippy Lab workflow

### PR DESCRIPTION
**Pull request by kushalbakshi**
This PR makes changes to `element-event` to run the Sippy Lab workflow. The `BehaviorTimeSeries` table is populated by the `ingestion.py` script within the Sippy Lab workflow. 

The table definition for `event_start_time` has been updated from `float` to `decimal` to avoid floating point precision errors and primary key errors in mysql.

### CODE REVIEW

**Review by ttngu207 — APPROVED** [review]

**Comment by kushalbakshi** [comment]
@ttngu207, I don't have write access in this repository either.

**Comment by kushalbakshi** [comment]
Self-merging based on offline discussion to close this out. After merging, the requirement will be updated in the workflow repository.

**Pull request merged**
